xcat-inventory/xcclient/allien/resources/nodehm.py

```diff
@@ -4,31 +4,47 @@
 
 # -*- coding: utf-8 -*-
 
-from flask import current_app
+import os
+from flask import current_app, request
 from flask_restplus import Resource, reqparse
-from ..hwmanager import *
+from .node import ns, actionreq
+from .service import ns as ns1
+from .service import srvreq
```

Comment:
please move the .node to the end of import,  normally the order is that `Sys -> flask -> parent -> current`